doc/configuration.rst

```diff
@@ -0,0 +1,87 @@
+Configuration
+-------------
+
```

Comment:
It should state at the top that you can change the settings within PuDB by pressing `Ctrl-p`. That's the most important thing here (and it isn't obvious to everyone).